python/paddle/fluid/layers/nn.py

```diff
@@ -12811,3 +12812,38 @@ def shard_index(input, index_num, nshards, shard_id, ignore_value=-1):
         },
         stop_gradient=True)
     return out
+
+
+@templatedoc()
+def hard_swish(x, threshold=6.0, scale=6.0, offset=3.0, name=None):
+    """
+    ${comment}
+    Args:
+        x(Varaible): Input of HardSwish operator.
```

Comment:
Input of HardSwish operator->Input of hard_swish layer

Reply:
This refers to the style of hard_sigmoid op, relu6 op, etc. Thanks~